pytx/pytx/common.py

```diff
@@ -411,6 +413,8 @@ def connections(cls_or_self,
         :type verify: bool, str
         :param metadata: Get extra metadata in the response.
         :type metadata: bool
+        :param summary: Get a summarized view of the connection.
+        :type summary: bool
```

Comment:
We will need to add `summary_param` below this so people will know what to pass in.